From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Thu, 29 Jun 2017 22:30:34 +0200
Subject: [PATCH] configure.ac: fix handling for --enable-clock_gettime

The logic for this configure option is broken; on ARM, we fall into the
*) path and the logic decides that this option is not set, no matter if
we set it or not.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index f8d76c106014..e4f17a80d876 100644
--- a/configure.ac
+++ b/configure.ac
@@ -175,6 +175,8 @@ case "$host" in
  *)
     if test "x$enable_clock_gettime" = "xx"; then
       enable_clock_gettime="yes"
+    elif test "x$enable_clock_gettime" = "xyes"; then
+      enable_clock_gettime="yes"
     else
       AC_MSG_ERROR([no timer source for host :"$host".])
     fi
